src/services/ui/src/features/package/hooks.tsx

```diff
@@ -16,19 +15,22 @@ export const useDetailsSidebarLinks = (
 
   useLayoutEffect(() => {
     const ids = [
-      "package_detailss",
-      "package_activity",
-      "administrative_package_changess",
+      { id: "package_details", label: "Package Details" },
+      { id: "package_activity", label: "Package Activity" },
+      {
+        id: "administrative_package_changes",
+        label: "Administrative Package Changes",
+      },
```

Comment:
I am wondering if this id -> label map should maybe be abstracted to shares utils? is similar code used anywhere else in the repo it just seems like something that might be?

This is outside the ticket itself so maybe something worth noting for the future. I will leave it to you to decide what would be best :)

Reply:
Oh you right I woulda sworn it showed up in more places 😅  
Nice work overall, you can ignore this lol